docs/structured-streaming-programming-guide.md

```diff
@@ -930,7 +930,7 @@ There are a few types of output modes.
 new rows added to the Result Table since the last trigger will be 
 outputted to the sink. This is supported for only those queries where 
 rows added to the Result Table is never going to change. Hence, this mode 
-guarantees that each row will be output only once (assuming 
+guarantees that each row will be outputted only once (assuming 
```

Comment:
Oh, hm. As a native speaker, I would have said "outputted" is incorrect actually. But it looks like it's an accepted alternative. Well, whatever's the smaller change to achieve consistency.